Packs/Core/Playbooks/playbook-Msiexec_execution_of_an_executable_from_an_uncommon_remote_location.yml

```diff
@@ -0,0 +1,1273 @@
+                    right:
+                      value:
+                        simple: alert.hostip
+                      iscontext: true
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": -105
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: bb15c39c-7d80-43d3-80ee-bb32ec6ce30f
+    type: regular
+    task:
+      id: bb15c39c-7d80-43d3-80ee-bb32ec6ce30f
+      version: -1
+      name: Check IP prevalence
+      description: Get the prevalence of an IP, identified by ip_address.
+      script: '|||core-get-IP-analytics-prevalence'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      ip_address:
+        complex:
+          root: IP.Address
+          filters:
+          - - operator: notIn
+              left:
+                value:
+                  simple: IP.Address
+                iscontext: true
+              right:
+                value:
+                  simple: alert.hostip
+                iscontext: true
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 710,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "45":
+    id: "45"
+    taskid: f8e02716-f9c2-4bb7-8464-8ff01abf0056
+    type: condition
+    task:
+      id: f8e02716-f9c2-4bb7-8464-8ff01abf0056
+      version: -1
+      name: Check if URL exist
+      description: Check if URL exist
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "25"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: URL.Data
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": -680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "1_43_#default#": 0.64,
+      "21_13_#default#": 0.54,
+      "45_25_yes": 0.52,
+      "45_3_#default#": 0.34,
+      "8_22_#default#": 0.51,
+      "8_42_Malicious": 0.48
+    },
+    "paper": {
+      "dimensions": {
+        "height": 3615,
+        "width": 1750,
+        "x": -550,
+        "y": -980
+      }
+    }
+  }
+inputs: []
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 8.8.0
```

Comment:
please add markteplacev2 for alignment purpose (i know the pack is xsiam only but it will make it easier when changing all playbooks in future)